sample.env

```diff
@@ -19,6 +19,7 @@ REACT_APP_FEATURE_PLUGIN_BITCOIN=false
 REACT_APP_FEATURE_PLUGIN_COSMOS=false
 REACT_APP_FEATURE_GEM_RAMP=false
 REACT_APP_REDUX_LOGGING=false
+REACT_APP_KEEP_KEY_SETTINGS=false
```

Comment:
```suggestion
REACT_APP_FEATURE_KEEPKEY_SETTINGS=false
```

Reply:
Renamed in https://github.com/shapeshift/web/pull/1341/commits/4ff5e2e7b910ed1038a6e6b9e6aca478245fb2af.